spec/coursemology/evaluator/models/base_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe Coursemology::Evaluator::Models::Base do
+  describe 'class attributes' do
+    subject { Coursemology::Evaluator::Models::Base }
+
+    describe '.api_user_email' do
+      it { is_expected.to have_attributes(api_user_email: nil) }
+    end
+
+    describe '.api_token' do
+      it { is_expected.to have_attributes(api_token: nil) }
+    end
+  end
+
+  describe '#adds_authentication' do
+    with_mock_client(api_user_email: 'email', api_token: 'token') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.